youtube_dl/extractor/zdf.py

```diff
@@ -70,6 +70,21 @@ def xml_to_format(fnode):
             '_available': is_available,
         }
 
+    def xml_to_thumbnails(fnode):
+        thumbnails = list()
+        for node in fnode:
+            thumbnail = {'url': node.text}
+            if 'key' in node.attrib:
+                if re.match("^[0-9]+x[0-9]+$", node.attrib['key']):
```

Comment:
Since you use regex anyway you can just capture both numbers and use plain `int`. Also consistently use single quotes.